File: node_backend/database/entities/PrevisionDataEntity.js
module.exports = class PrevisionDataEntity {
	constructor(previsionDataId, previsionId, arriveTime, waitSeconds) {
		this.previsionDataId = previsionDataId;
		this.previsionId = previsionId;
		this.arriveTime = arriveTime;
		this.waitSeconds = waitSeconds;
	}
}

File: queue_prediction/db/prevision_data_entity.py
#       __  __     _______ ______
#     / / / /__  / __/ _ /_  __/__
#   / /_/ / _ \/ _// __ |/ / / _ \
#  \____/_//_/___/_/ |_/_/ /_//_/
#
#  Author: Willi Menapace <willi.menapace@gmail.com>
#

import datetime

#Wait time prevision data
class PrevisionDataEntity:

    previsionDataId = None
    previsionId = None
    arriveTime = datetime.datetime.now().time()
    waitTimeSeconds = 0

    def __init__(self, previsionDataId, previsionId, arriveTime, waitTimeSeconds):
        self.previsionId = previsionId
        self.arriveTime = arriveTime
        self.waitTimeSeconds = waitTimeSeconds
